<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>815</width>
    <height>646</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>801</width>
      <height>571</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="posWidget">
     <attribute name="title">
      <string>Players</string>
     </attribute>
     <widget class="QTabWidget" name="posTab">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>0</y>
        <width>800</width>
        <height>525</height>
       </rect>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="newInvoiceTab">
       <attribute name="title">
        <string>Update Stats</string>
       </attribute>
       <widget class="QGroupBox" name="newInvoiceGroupBox">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>800</width>
          <height>525</height>
         </rect>
        </property>
        <property name="title">
         <string/>
        </property>
        <widget class="QLabel" name="label_7">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>30</y>
           <width>60</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string>Name</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_8">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>70</y>
           <width>60</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string>Goals</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_9">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>110</y>
           <width>60</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string>Assists</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_10">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>150</y>
           <width>60</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string>Points</string>
         </property>
        </widget>
        <widget class="QPushButton" name="updateButtonEditStatsTab">
         <property name="geometry">
          <rect>
           <x>90</x>
           <y>200</y>
           <width>260</width>
           <height>41</height>
          </rect>
         </property>
         <property name="text">
          <string>Update!</string>
         </property>
        </widget>
        <widget class="QComboBox" name="nameComboBoxUpdateStatsTab">
         <property name="geometry">
          <rect>
           <x>90</x>
           <y>30</y>
           <width>260</width>
           <height>20</height>
          </rect>
         </property>
        </widget>
        <widget class="QTableWidget" name="invoiceListTableWidgetNewInvoiceTab">
         <property name="geometry">
          <rect>
           <x>390</x>
           <y>30</y>
           <width>400</width>
           <height>400</height>
          </rect>
         </property>
        </widget>
        <widget class="QLabel" name="feedbackLabelNewInvoiceTab">
         <property name="geometry">
          <rect>
           <x>90</x>
           <y>310</y>
           <width>260</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
        <widget class="QLineEdit" name="pointsLineEditUpdateStatsTab">
         <property name="geometry">
          <rect>
           <x>90</x>
           <y>150</y>
           <width>260</width>
           <height>20</height>
          </rect>
         </property>
        </widget>
        <widget class="QLineEdit" name="AssistsLineEditUpdateStatsTab">
         <property name="geometry">
          <rect>
           <x>90</x>
           <y>110</y>
           <width>260</width>
           <height>20</height>
          </rect>
         </property>
        </widget>
        <widget class="QLineEdit" name="goalsLineEditUpdateStatsTab">
         <property name="geometry">
          <rect>
           <x>90</x>
           <y>70</y>
           <width>260</width>
           <height>20</height>
          </rect>
         </property>
        </widget>
        <widget class="QLineEdit" name="idLineEditUpdateStatsTab">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="geometry">
          <rect>
           <x>90</x>
           <y>370</y>
           <width>260</width>
           <height>20</height>
          </rect>
         </property>
        </widget>
       </widget>
      </widget>
      <widget class="QWidget" name="addCustomerTab">
       <attribute name="title">
        <string>Add Player</string>
       </attribute>
       <widget class="QGroupBox" name="addCustomerGroupBox">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>800</width>
          <height>525</height>
         </rect>
        </property>
        <property name="title">
         <string/>
        </property>
        <widget class="QLabel" name="label">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>30</y>
           <width>60</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string>First Name</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="firstNameLineEditAddPlayerTab">
         <property name="geometry">
          <rect>
           <x>90</x>
           <y>30</y>
           <width>260</width>
           <height>20</height>
          </rect>
         </property>
        </widget>
        <widget class="QLabel" name="label_2">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>70</y>
           <width>60</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string>Last Name</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="lastNameLineEditAddPlayerTab">
         <property name="geometry">
          <rect>
           <x>90</x>
           <y>70</y>
           <width>260</width>
           <height>20</height>
          </rect>
         </property>
        </widget>
        <widget class="QLineEdit" name="positionLineEditAddPlayerTab">
         <property name="geometry">
          <rect>
           <x>90</x>
           <y>110</y>
           <width>260</width>
           <height>20</height>
          </rect>
         </property>
        </widget>
        <widget class="QLabel" name="label_3">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>110</y>
           <width>60</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string>Position</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_4">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>150</y>
           <width>60</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string>Number</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="numberLineEditAddPlayerTab">
         <property name="geometry">
          <rect>
           <x>90</x>
           <y>150</y>
           <width>260</width>
           <height>20</height>
          </rect>
         </property>
        </widget>
        <widget class="QLineEdit" name="countryLineEditAddPlayerTab">
         <property name="geometry">
          <rect>
           <x>90</x>
           <y>190</y>
           <width>260</width>
           <height>20</height>
          </rect>
         </property>
        </widget>
        <widget class="QLabel" name="label_5">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>190</y>
           <width>60</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string>Country</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_6">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>310</y>
           <width>261</width>
           <height>20</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">color: red;</string>
         </property>
         <property name="text">
          <string>PLayer ID will be automatically generated</string>
         </property>
        </widget>
        <widget class="QPushButton" name="addPlayerButtonAddPlayerTab">
         <property name="geometry">
          <rect>
           <x>90</x>
           <y>240</y>
           <width>260</width>
           <height>31</height>
          </rect>
         </property>
         <property name="text">
          <string>Add Player</string>
         </property>
        </widget>
        <widget class="QLabel" name="feedbackLabelAddCustomerTab">
         <property name="geometry">
          <rect>
           <x>90</x>
           <y>300</y>
           <width>260</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </widget>
      </widget>
      <widget class="QWidget" name="editCustomerTab">
       <attribute name="title">
        <string>Edit Player</string>
       </attribute>
       <widget class="QGroupBox" name="editPlayerGroupBox">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>0</y>
          <width>811</width>
          <height>521</height>
         </rect>
        </property>
        <property name="title">
         <string/>
        </property>
        <widget class="QLabel" name="label_12">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>30</y>
           <width>60</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string>Name</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_13">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>70</y>
           <width>60</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string>ID</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="idNameLineEditEditPlayerTab">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="geometry">
          <rect>
           <x>90</x>
           <y>70</y>
           <width>260</width>
           <height>20</height>
          </rect>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
        <widget class="QLineEdit" name="postionLineEditEditPLayerTab">
         <property name="geometry">
          <rect>
           <x>90</x>
           <y>110</y>
           <width>260</width>
           <height>20</height>
          </rect>
         </property>
        </widget>
        <widget class="QLabel" name="label_14">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>110</y>
           <width>60</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string>Position</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_15">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>150</y>
           <width>60</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string>Number</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="numberLineEditEditPlayerTab">
         <property name="geometry">
          <rect>
           <x>90</x>
           <y>150</y>
           <width>260</width>
           <height>20</height>
          </rect>
         </property>
        </widget>
        <widget class="QLineEdit" name="countryLineEditEditPlayerTab">
         <property name="geometry">
          <rect>
           <x>90</x>
           <y>190</y>
           <width>260</width>
           <height>20</height>
          </rect>
         </property>
        </widget>
        <widget class="QLabel" name="label_16">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>190</y>
           <width>60</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string>Country</string>
         </property>
        </widget>
        <widget class="QPushButton" name="saveChangesButton">
         <property name="geometry">
          <rect>
           <x>90</x>
           <y>310</y>
           <width>261</width>
           <height>31</height>
          </rect>
         </property>
         <property name="text">
          <string>Save Changes</string>
         </property>
        </widget>
        <widget class="QLabel" name="feedbackLabelAddCustomerTab_2">
         <property name="geometry">
          <rect>
           <x>90</x>
           <y>300</y>
           <width>260</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
        <widget class="QComboBox" name="nameComboBoxEditPlayerTab">
         <property name="geometry">
          <rect>
           <x>90</x>
           <y>30</y>
           <width>260</width>
           <height>20</height>
          </rect>
         </property>
        </widget>
        <widget class="QLabel" name="feedbackLabelEditCustomerTab">
         <property name="geometry">
          <rect>
           <x>89</x>
           <y>370</y>
           <width>251</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
        <widget class="QLabel" name="label_17">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>230</y>
           <width>60</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string>First Name</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_18">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>270</y>
           <width>60</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string>Last Name</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="firstNameLineEditPlayerTab">
         <property name="geometry">
          <rect>
           <x>90</x>
           <y>230</y>
           <width>260</width>
           <height>20</height>
          </rect>
         </property>
        </widget>
        <widget class="QLineEdit" name="lastNameLineEditEditPlayerTab">
         <property name="geometry">
          <rect>
           <x>90</x>
           <y>270</y>
           <width>260</width>
           <height>20</height>
          </rect>
         </property>
        </widget>
        <widget class="QPushButton" name="removePlayerEditPlayerTabButton">
         <property name="geometry">
          <rect>
           <x>90</x>
           <y>360</y>
           <width>261</width>
           <height>31</height>
          </rect>
         </property>
         <property name="text">
          <string>Remove Player</string>
         </property>
        </widget>
       </widget>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="viewWidget">
     <attribute name="title">
      <string>View</string>
     </attribute>
     <widget class="QGroupBox" name="viewGroupBox">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>781</width>
        <height>541</height>
       </rect>
      </property>
      <property name="title">
       <string/>
      </property>
      <widget class="QComboBox" name="viewSelectionComboBoxViewTab">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>40</y>
         <width>671</width>
         <height>22</height>
        </rect>
       </property>
       <item>
        <property name="text">
         <string>Inventory</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Out Of Stock</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Vendors</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Customers</string>
        </property>
       </item>
      </widget>
      <widget class="QLabel" name="label_33">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>40</y>
         <width>81</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>View Selection</string>
       </property>
      </widget>
      <widget class="QTableWidget" name="viewTableWidgetViewTab">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>70</y>
         <width>751</width>
         <height>461</height>
        </rect>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>815</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
